app/javascript/components/batch_update/components/App.js

```diff
@@ -0,0 +1,640 @@
+/* eslint-disable @typescript-eslint/no-this-alias */
```

Comment:
Woah! did you write all this JS?
Or Is it automatically generated?

Reply:
oh! yeah, everything on App.js was automagically generated (I don't even know at what point this happened 🤔 )

Reply:
oh I just found out! I have it generated on main when I run `yarn tsc` (so I think it makes sense to have it versioned?)

Reply:
Ahh i need to look closer at things. If its not already in the repo, i'd remove it. I think what happened is that tsc also compiled the files (where elsewhere at artsy we only use it as a typechecker). I imagine that's what you were trying to do? fwiw, `yarn tsc --noEmit`

We should prob delete this as well as the .map file.

Reply:
Thank you both for catching this! Opening a PR for removing it [here](https://github.com/artsy/rosalind/pull/790)

Reply:
Ah yeah, we [began](https://github.com/artsy/rosalind/pull/672) to add Typescript support in 2022  (to this ca. 2016 app!) when we thought we were going to do a round of work in it, but then our roadmap changed so it's possible there are some rough edges in the TS support here. Thanks for catching that 🙏🏽

Reply:
🙏🏻  thanks all!